stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/AttachmentFactory.kt

```diff
@@ -45,5 +44,7 @@ public open class AttachmentFactory constructor(
         modifier: Modifier,
         attachmentState: AttachmentState,
     ) -> Unit,
-    public val textFormatter: (attachments: Attachment) -> String = Attachment::previewText,
+    public val textFormatter: (attachments: Attachment) -> String = {
+        it.title ?: it.name ?: ""
+    },
```

Comment:
Why don't we use the extension?

Reply:
It was the last extension in the file and it was used only in one place, so I decided to move the code to the call site.